### PR TITLE
Remove get alert rule

### PR DESCRIPTION
**Pull request by johntdyer**
Minor fix for a few struct keys

### CODE REVIEW

File: dns_server.go [hunk 1]
```diff
@@ -38,7 +38,8 @@ type DNSServer struct {
 	Interval              int            `json:"interval,omitempty"`
 	MtuMeasurements       int            `json:"mtuMeasurements,omitempty"`
 	NetworkMeasurements   int            `json:"networkMeasurements,omitempty"`
-	NumPathTraces         int            `json:"numPathTraces,omitempty"`
+	NumPathTraces         int            `json:"numPathTraces"`
```
Comment (william20111) [inline dns_server.go:41]: What does the omitempty need to leave for on this? Can it be a valid 0?

Comment (william20111) [reply dns_server.go:41]: https://developer.thousandeyes.com/v6/tests/#/test_metadata looks like its 1..10 everywhere?

Comment (johntdyer) [reply dns_server.go:41]: yea, so I guess it doesnt really matter either way since it cant ever be empty... want me to remove it  ?

Comment (william20111) [reply dns_server.go]: It’s fine, just leave it will merge it can’t be empty and has a default so it’s fine for now

**Pull request merged**